app/controllers/users_controller.rb

```diff
@@ -0,0 +1,35 @@
+class UsersController < ApplicationController
+  def edit
+    @user = User.find(params[:id])
+  end
+
+  def destroy
+    @user = User.find(params[:id])
+    @user.destroy
+    flash[:notice] = "Account successfully deleted!"
+    redirect_to root_path
+  end
+
+  def update
+    @user = User.find(params[:id])
+    @email = params[:user][:email]
```

Comment:
Šādi tev būs reti jādara, jo tev parametrus jau izfiltrē tās user_params metodes ar params(:require).permit(....)

Beigās tev paliek kkas Hash līdzīgs un tu pa lielam vari taisīt  @user.update(user_params)

Reply:
oh , es nezinaju ka ta funkcija tā atgriez vertibas , nekadigi nevaru pierast ar to ka return nav. Uzreiz lecu spriest ka kkas backgroundā izsauks to funkciju. Tgd safixoju.